bin/Print.ml

```diff
@@ -0,0 +1,53 @@
+open Odiff.Diff
+
+let printDiffResult makeParsableOutput result =
+  (match (result, makeParsableOutput) with
+  | Layout, true -> ""
+  | Layout, false ->
+      Pastel.createElement
+        ~children:
+          [
+            (Pastel.createElement ~color:Red ~bold:true ~children:[ "Failure!" ]
+               () [@JSX]);
+            " Images have different layout.\n";
+          ]
+        () [@JSX]
+  | Pixel (_output, diffCount, _percentage, _lines), true when diffCount == 0 ->
+      ""
+  | Pixel (_output, diffCount, _percentage, _lines), false when diffCount == 0
+    ->
+      Pastel.createElement
+        ~children:
+          [
+            (Pastel.createElement ~color:Green ~bold:true
+               ~children:[ "Success!" ] () [@JSX]);
+            " Images are equal.\n";
+            (Pastel.createElement ~dim:true
+               ~children:[ "No diff output created." ]
+               () [@JSX]);
+          ]
+        () [@JSX]
+  | Pixel (_output, diffCount, diffPercentage, stack), true
+    when not (Stack.is_empty stack) ->
+      Int.to_string diffCount ^ ";"
```

Comment:
Might be easier to use `Prinf.sprintf "%d;%f;%s" diffCount diffPercentage (Stack.fold ...)`

Reply:
I prefer do all the formatting using one library, probably would be good to migrate to spectrum or else but in the other PR